### PR TITLE
websocket support

### PR DESCRIPTION
**Pull request by indatawetrust**
I noticed that the package does not support websocket. I wanted to make and share the development because I needed it. I am waiting for your comments. Thanks.

### CODE REVIEW

File: template.conf [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
     {% if options.https %}
       ssl_protocols TLSv1.1 TLSv1.2;
-      ssl_certificate {{ settings.conf }}/../certs/{{ site.shortname }}.cer;
-      ssl_certificate_key {{ settings.conf }}/../certs/{{ site.shortname }}.key;
+      ssl_certificate /etc/letsencrypt/live/{{ site.host }}/fullchain.pem;
```
Comment (abea) [inline template.conf]: Do these letsencrypt changes have something to do with the websocket functionality? they should probably be reverted.

Comment (indatawetrust) [reply template.conf]: nothing to do with websocket, i fixed it

Comment (abea) [reply template.conf]: Thanks. There's the additional lines 21 and 22 with the same change. @boutell will be able to give a proper review of the core changes with that reverted.

**Comment by boutell** [comment]
Thanks!

**Comment by boutell** [comment]
One change made (and published as 1.3.1): we went with `--websockets`, not `--websocket`, because everyone refers to websockets in the plural.

**Pull request merged**